FlashDevelop/Bin/Debug/Projects/310 Haxe - Flash Project/Project.hxproj

```diff
@@ -8,7 +8,7 @@
     <movie fps="30" />
     <movie width="800" />
     <movie height="600" />
-    <movie version="17" />
+    <movie version="18" />
```

Comment:
Last supported version for haxe 3.2 is 17 = 28. 3.1.3 supports also up to 12.5 -> 28 but swf-version is labeled incorrectly. Last correct version is 12.0 -> 23. What should we do?